packages/components/src/composite/index.ts

```diff
@@ -1,4 +1,8 @@
-// Until we migrate away from Reakit, the 'unstable'
-// implementation remains the default.
+// Originally this pointed at a Reakit implementation of
+// `Composite`, but we are removing Reakit entirely from the
+// codebase. Until the new 'current' Ariakit implementation
+// is unlocked and no longer private, we will continue to
+// support the Reakit API through the 'legacy' version,
+// which uses Ariakit under the hood.
```

Comment:
We will continue to support this older implementation also after the new `Ariakit` implementation is unlocked, at least for a while

```suggestion
// codebase. We will continue to support this older implementation
// through a legacy version which uses Ariakit under the hood.
```